RecoTracker/LSTCore/standalone/code/core/write_lst_ntuple.cc

```diff
@@ -579,6 +621,146 @@ void setPixelTripletOutputBranches(lst::Event<Acc3D>* event) {
   ana.tx->setBranch<std::vector<int>>("pT3_isDuplicate", pT3_isDuplicate);
 }
 
+//________________________________________________________________________________________________________________________________
+void fillT5DNNBranches(lst::Event<Acc3D>* event, unsigned int T3) {
+  // Get relevant information
+  lst::Hits const* Hits = event->getHits()->data();
```

Comment:
```suggestion
  lst::Hits const* hits = event->getHits()->data();
```
lower case for local names; caps are for types (same comment for the rest of the PR).

Reply:
Do I have to use camelCase for all variables or are underscores fine? The write_lst_ntuple.cc code was taken from Jonathon's code btw.

Reply:
I think the snake_case is OK